tests/test_data.py

```diff
@@ -34,6 +34,8 @@ def test_fuzzy_finder(keywords, paper_id):
     "ptsd",
     "ace",
     "hall",
+    "benchmark:Bos_2018",
+    "benchmark:van_de_Schoot_2017"
```

Comment:
Do we need to add more of such data for testing purposes?

Reply:
Maybe we should test all the featured datasets.